src/carpadi_admin/views.py

```diff
@@ -103,3 +106,11 @@ class ActivityViewSetAdmin(viewsets.ReadOnlyModelViewSet):
     queryset = Activity.objects.all()
     filter_backends = (filters.DjangoFilterBackend,)
     filter_class = ActivityFilterAdmin
+
+
+class SparePartsViewSet(viewsets.ModelViewSet):
+    permissions = {'default': (IsAdminUser,)}
```

Comment:
use `permission_classes = (IsAdminUser, )` instead of this. this wont work in this case. did you test it, if so how did you test it?

Reply:
I actually followed the way it was for other resources.

Reply:
then they wont work too. that is not how to use it if you want to use it like that

Reply:
Should I change it for all resources?

Reply:
yes and no. the stuff requires a method in the view for it to work. if that method is present it will work and even useful. the issue is that the one you declared and the other ones dont have that function hence why they wont work.
this is the function `src.carpadi_api.views.TransactionViewSet.get_permissions` so if you can copy that function to those views. then we should be good.

Reply:
I updated the code, please at the latest commit: Line 114, 115. Let me know which one works